back/app/models/idea.rb

```diff
@@ -71,12 +71,13 @@ class Idea < ApplicationRecord
   has_many :baskets_ideas, dependent: :destroy
   has_many :baskets, through: :baskets_ideas
   has_many :text_images, as: :imageable, dependent: :destroy
-  accepts_nested_attributes_for :text_images
 
-  has_many :idea_images, -> { order(:ordering) }, dependent: :destroy
+  has_many :idea_images, -> { order(:ordering) }, dependent: :destroy, inverse_of: :idea
```

Comment:
Why remove the default order?

Reply:
Does it remove default order ? I only added inverse_of

Reply:
Does it remove default order ? I only added inverse_of

Reply:
Haha, yes, seems like was blind :man_facepalming: